### PR TITLE
feat(oncall): add the request_id to the query breadcrumbs

### PR DESCRIPTION
**Pull request by volokluev**
Searching for queries by id in the querylog requires the request_id because that is the `orderBy` key for the table. Without it, the queries timeout. This assures that any query that makes it to sentry has a request_id attached to it

### CODE REVIEW

**Review by lynnagara — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -361,6 +361,7 @@ jobs:
             -e REDIS_HOST=sentry_redis \
             -e REDIS_PORT=6379 \
             -e REDIS_DB=1 \
+            -e ENABLE_AUTORUN_MIGRATION_SEARCH_ISSUES=1 \
```
Comment (evanh) [inline .github/workflows/ci.yml:364]: Why this change?

Comment (volokluev) [reply .github/workflows/ci.yml:364]: It was breaking CI to not have it. It's in master now
